src/library_gl.js

```diff
@@ -1069,52 +1073,74 @@ var LibraryGL = {
 
   glCompressedTexImage2D__sig: 'viiiiiiii',
   glCompressedTexImage2D: function(target, level, internalFormat, width, height, border, imageSize, data) {
-#if ASSERTIONS
-    assert(GL.currentContext.compressionExt);
```

Comment:
I see.

I am probably just not qualified to review this code ;) I'll go over it as best I can, but if you feel you need a serious review of the GL aspects, we should probably ask for an additional reviewer (WebGL2 dev?)